cmd/ignite/run/inspect.go

```diff
@@ -0,0 +1,61 @@
+package run
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/weaveworks/ignite/pkg/apis/ignite/scheme"
+	"github.com/weaveworks/ignite/pkg/client"
+	"github.com/weaveworks/ignite/pkg/filter"
+
+	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
+)
+
+type InspectFlags struct {
+	YAMLOutput bool
+}
+
+type inspectOptions struct {
+	*InspectFlags
+	object meta.Object
+}
+
+func (i *InspectFlags) NewInspectOptions(k, objectMatch string) (*inspectOptions, error) {
+	var err error
+	var kind meta.Kind
+	io := &inspectOptions{InspectFlags: i}
+
+	switch k {
```

Comment:
`strings.ToLower(k)` to support `ignite inspect Image foo` if somebody would do it. no need to error in that case

Reply:
Yes, was not sure if it's needed as it could train the user "incorrect" usage. Implemented in c6622cf now.